scoreserver/go.mod

```
@@ -0,0 +1,27 @@
+module kfd
```

モジュール名はgithub.comから見たディレクトリ名にしておくと良いです
```suggestion
module github.com/uta8a/kfd/scoreserver
```

---

ありがとうございます！

memo: ローカル完結ならURL形式でなくても良いが、githubにあげるならgithub.com始まりが良い